assets/src/stories-editor/plugins/context-menu-item.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Plugin that adds a remove page button to the "More" settings menu.
+ */
+
+/**
+ * External dependencies
+ */
+import { ReactElement } from 'react';
```

Comment:
Is this used? And if it is, shouldn't it be an import from `@wordpress/element`?

Reply:
That was a copy and paste from somewhere else in the code base. I don't love importing but I got a warning in the lints if I didn't have it.